frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -550,6 +542,17 @@ class Visualization extends PureComponent {
   }
 }
 
+function checkIsLoading(series, props) {
```

Comment:
maybe call it `isLoadingOrEmpty` to avoid overloading `isLoading` term

Reply:
Agree with Alex P